libevmasm/Assembly.cpp

```diff
@@ -49,6 +49,8 @@ void Assembly::append(Assembly const& _a)
 	}
 	m_deposit = newDeposit;
 	m_usedTags += _a.m_usedTags;
+	// This does not append the named tags on purpose, they are only available by name inside
+	// the assembly.
```

Comment:
Hm? I think it does. The tag for a named tag is just `newTag()` == `m_usedTags++`.

Reply:
Ok, let me rephrase to "does not retain the names of named tags on purpose (the tags themselves are of course carried over)"